src/items/item.cpp

```diff
@@ -974,51 +974,65 @@ uint32_t Item::getWeight() const
 	return weight;
 }
 
-std::vector<std::pair<std::string, std::string>>
-											Item::getDescriptions(const ItemType& it, const Item* item /*= nullptr*/) {
+std::vector<std::pair<std::string, std::string>> Item::getDescriptions(const ItemType& it, const Item* item /*= nullptr*/) {
 	std::ostringstream ss;
 	std::vector<std::pair<std::string, std::string>> descriptions;
+	bool isTradeable = true;
 	descriptions.reserve(30);
 	if (item) {
 		const std::string& specialDescription = item->getSpecialDescription();
 		if (!specialDescription.empty()) {
-        descriptions.emplace_back("Description", specialDescription);
-    } else if (!it.description.empty()) {
```

Comment:
Are you sure this can be removed? Unless mistakenly, this is if performs the function when the player does not have the item, such as looks in the NPC trade (where the player does not have the item but needs to give look anyway)